tests/cephadm/test_ceph_health_detail_when_mon_offline.py

```diff
@@ -0,0 +1,73 @@
+from time import sleep
+
+from cli.cephadm.cephadm import CephAdm
+from cli.utilities.utils import bring_node_offline, is_node_online
+
+
+class ClusterNotHealthy(Exception):
+    pass
+
+
+class NodeNotOffline(Exception):
```

Comment:
Can be changed to common exception `UnexpectedNodeStatusError`

Reply:
Changes made